main.js

```diff
@@ -153,6 +165,18 @@ document.querySelector('[data-view="confirmation"]').addEventListener('click', f
   }
 })
 
+function sort(sortBy) {
+  sortedList = appState.catalog.items.sort(function (obj1, obj2) {
```

Comment:
This function should take an unsorted list and return a sorted one. Avoid polluting the global scope with extra variables.